js/renovation/viz/core/common/tooltip_utils.ts

```diff
@@ -50,8 +50,8 @@ export function getCloudPoints(
   const anchorX = rotateX(coordinates, radRotationAngle);
   const anchorY = rotateY(coordinates, radRotationAngle);
   const halfArrowWidth = options.arrowWidth / 2;
-  const halfWidth = width / 2;
-  const halfHeight = height / 2;
+  const halfWidth = Number(width) / 2;
+  const halfHeight = Number(height) / 2;
```

Comment:
Why is there `Number`? width and height type is number anyway

Reply:
https://trello.com/c/oSnG9nzt/1360-remove-undefined-operator-from-tooltip-props